stable/metrics-server/templates/cluster-role.yaml

```diff
@@ -19,4 +19,11 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+    - ""
+    resources:
+    - nodes/stats
+    verbs:
+    - get
+    - create
```

Comment:
This seems not to be the same upstream (https://github.com/kubernetes-incubator/metrics-server/blob/master/deploy/1.8%2B/resource-reader.yaml)
Also, there's some stuff in there related to `Deployment`s. Should this be synced?

Reply:
I'll check that out, thanks. I did see error messages related to `nodes/stats:create` but I'll check again using the upstream ClusterRole. I agree that this chart should probably install the same ClusterRole as upstream.

Reply:
The PR in the current state seems to be the most correct one.

The deployment part is not needed. Upstream only has it due to pod-nanny. The upstream ClusterRole has the two rules merged, but that means more verbs than strictly necessary for the stats resource.

Reply:
OK, then I will leave things as they are, and we just need to wait for the **pull-charts-e2e** and **tide** checks to go green.